docker/xenial-proposed.Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:xenial
+
+# Enable proposed and pin snapcraft
+RUN echo "deb http://archive.ubuntu.com/ubuntu/ xenial-proposed restricted main multiverse universe" >> /etc/apt/sources.list
+RUN cat /etc/apt/sources.list
+RUN echo 'Package: *' > /etc/apt/preferences.d/snapcraft-proposed
+RUN echo 'Pin: release a=xenial-proposed' >> /etc/apt/preferences.d/snapcraft-proposed
+RUN echo 'Pin-Priority: 400' >> /etc/apt/preferences.d/snapcraft-proposed
+
+RUN apt-get update && \
+  apt-get dist-upgrade --yes && \
+  apt-get install --yes \
+  git \
+  snapcraft/xenial-proposed \
+  && \
+  apt-get autoclean --yes && \
+  apt-get clean --yes
+
+# Required by click.
+ENV LC_ALL C.UTF-8
```

Comment:
This is no longer technically necessary.

Reply:
fi not set, you get a warning, so I'd say we leave it as we do not silently set it.